.github/workflows/build-rd-cpp.yml

```diff
@@ -0,0 +1,234 @@
+name: rd-cpp
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  NINJA_VERSION: 1.9.0
+  BUILD_TYPE: Release
+  MAJOR_VERSION: 201.1
+
+jobs:
+  build:
+    env:
+      working-dir: ${{ github.workspace}}${{ matrix.config.SEP }}rd-cpp
+      version: 201.1.${{ github.run_number }}
+      artifact: ${{ matrix.config.artifact }}201.1.${{ github.run_number }}.zip
+    name: ${{ matrix.config.name }}
+    runs-on: ${{ matrix.config.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+        - {
+            name: "RD CPP Windows Latest MSVC", artifact: "RD-CPP-Windows-MSVC-2019-",
+            os: windows-latest,
+            cc: "cl", cxx: "cl",
+            environment_script: "C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Auxiliary/Build/vcvars64.bat",
+            SEP: \
+          }
+        - {
+            name: "RD CPP Windows VS 2017", artifact: "RD-CPP-Windows-MSVC-2017-",
+            os: windows-2016,
+            cc: "cl", cxx: "cl",
+            environment_script: "C:/Program Files (x86)/Microsoft Visual Studio/2017/Enterprise/VC/Auxiliary/Build/vcvars64.bat",
+            SEP: \
+          }
+        - {
```

Comment:
I suggest to disable clang, and only merge these two configurations once they become green.